demos/_unit-test/lookup.php

```diff
@@ -20,4 +20,13 @@
 };
 
 $crud = Crud::addTo($app);
+
+$crud->onFormEdit(function ($f) {
```

Comment:
tests are still failing

I think we should not need this, as I override this for testing **globally** in:
https://github.com/atk4/ui/blob/72e19a60edd150141890e6631276f0925da3449c/tests-behat/bootstrap/Context.php#L62

PS: I already tried to set `none` - without any change

if dropdown is loaded from JS and you do more than one operation in step, you should probably call `jqueryWait` between them

Reply:
It is still possible that Dropdown module override delay via js.

Reply:
yes, but try locally (where you can see) - for me, it seems this disables the animations